site/ko/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,175 @@
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/66878d6044df77ba6264a576483bf5aae6b5f3d9 #}
+
+Chromium issue: [1054464](https://crbug.com/1054464)
+
+
+## 다가오는 콘솔 사이드바의 디프리케이션 {: #deprecated }
+
+필터 사용자 인터페이스를 툴바로 옮기기 위해, 콘솔 사이드바가 더 이상 사용되지 않고 제거될 예정입니다. 혹시 염려되는 게 있으시거나 어떤 피드백이 있으시다면 [이슈 트래커](https://crbug.com/1232937)를 통해 알려 주세요.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/CzC2HCaiCcdPgbLykyc8.png", alt="Console sidebar deprecation message", width="800", height="474" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f34c62f543c29ffd4be95c4e93b453aa34644897 #}
+
+Chromium issue: [1232937](https://crbug.com/1232937)
+
+
+## 이슈 탭과 네트워크 패널에서 로우(raw) Set-Cookie 헤더 표시 {: #raw-cookies }
+
+DevTools 의 **이슈** 탭에서 이제 로우(raw) ```Set-Cookie``` 헤더를 볼 수 있습니다.
+
+이전에는 쿠키가 형식에 맞지 않았을 때 (정확하지 않은 ```Set-Cookie``` 헤더),  이는 DevTools의 네트워크 패널에 표시되지 않았습니다. 한편 새롭게 ```response-header-set-cookie``` 필터가 **네트워크** 패널에 추가됨으로써, 이제 사용자들은 로우(raw) ```Set-Cookie``` 헤더 응답을 걸러낼 수 있습니다. DevTools 는 **이슈** 탭의 로우(raw) ```Set-Cookie``` 헤더 응답을 **네트워크** 패널로 연결합니다.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/PbozcNJRd6rTME5hhqIq.png", alt="Raw 'Set-Cookie' headers in the Issues tab and Network panel", width="800", height="563" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/6dedde59f9d64290756a826f73dfe24cf382a470 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/27aa364d1b194a7a778e7649e1f144abbed5957f #}
+
+Chromium issue: [1179186](https://crbug.com/1179186)
+
+
+## 고유한 프로퍼티로서의 내이티브 접근자를 콘솔에서 일관성 있게 표시 {: #native-accessors }
+
+**콘솔**에서 내이티브 접근자를 그것의 고유한 프로퍼티로서 일관성 있게 표시할 수 있습니다.
+
+예를 들면 **콘솔**에서 `new Int8Array([1, 2, 3])` 코드를 표시할 때,  ```length```, ```byteOffset``` 과 같은 내이티브 접근자들은 미리보기에서 표시되지 않았습니다만, 이번 업데이트를 통해 사용자들은 내이티브 접근자를 미리보기에서 확인할 수 있으며 확장 시에 값들이 빠르게 표시됩니다.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/VcUiEcUXdWc00Q8595n6.png", alt="Consistent display native accessors as own properties in the Console", width="800", height="459" %}
+
+{# https://chromium.googlesource.com/v8/v8/+/ce8cef36aa7f386937a6b7bf1907e93b69cad1bd #}
+
+Chromium issues: [1076820](https://crbug.com/1076820), [1199247](https://crbug.com/1199247)
+
+
+## #sourceURL을 포함하는 인라인 스크립트를 위한 적절한 오류 스택 추적 {: #inline-script }
+
+DevTools 는 현재 ```#sourceURL``` 이 있는 인라인 스크립트를 잘 해결하고 있으며, 또한 적절한 오류 스택 추적을 제공하여 사용자의 디버깅을 돕습니다.
+
+이전의 DevTools 는 `#sourceURL`이 있는 인라인 스크립트에 대해 잘못된 위치를 표시했으며, 이는 ```<script>``` 의 시작점이 아닌 주변의 문서가 기준이 되었습니다.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XVUY8XxbGZW74kPsGkOZ.png", alt="Proper error stack traces for inline scripts with #sourceURL", width="800", height="425" %}
+
+{# https://chromium.googlesource.com/v8/v8/+/c2f30c2b3f637c2339e8b9672c5c59a21b7d1095 #}
+
+Chromium issues: [1183990](https://crbug.com/1183990), [578269](https://crbug.com/578269)
+
+## 계산된 창에서의 색상 포맷 변경 {: #color-unit }
+
+색상 미리보기에서 <kbd>Shift</kbd> 와 함께 클릭함으로써 사용자들은 이제 계산된 창에 있는 어떤 엘레먼트들의 색상 포맷을 변경할 수 있습니다.
```

Comment:
색상 미리보기를 Shift 키를 누른 채 클릭하여,
계산됨 영역에서 요소의 컬러 포맷을 변경할 수 있습니다.

Reply:
Would like to know which is correct one, 계산됨 영역 vs 계산된 영역. @techhtml

Reply:
because '계산됨' is the area of DevTools.
I think it need to match with DevTools translation.

![image](https://user-images.githubusercontent.com/6267393/133388613-433d0c85-f088-47ac-94c8-c46081734f56.png)

Reply:
Got it, will update. thanks